test/testsymboldatabase.cpp

```diff
@@ -42,11 +42,13 @@ struct InternalError;
 #define GET_SYMBOL_DB(code) \
     Tokenizer tokenizer(&settings1, this); \
     const SymbolDatabase *db = getSymbolDB_inner(tokenizer, code, "test.cpp"); \
-    ASSERT(db);
+    ASSERT(db); \
```

Comment:
Why not just removing the semicolon after the `ASSERT`?

Reply:
Seems a bit too subtle to me. It's obviously the same as `do {} while(false}`, but at least that's a bit more obvious and seem to be a common pattern for this.

I haven't seen one for forcing the semicolon for a macro which creates scope stuff yet - mostly it seems fine to just add the redundant one or omit completely.